docs/user-guide/glossary.md

```diff
@@ -79,6 +106,21 @@ scene
   The *scene* refers to the set of contract instances that the Prover knows
   about.
 
+SMT
+SMT solver
+  "SMT" is short for "Satisfiability Modulo Theory". An SMT solver takes as 
+  input a formula in predicate logic and returns whether the formula is 
+  satisfiable (short "SAT") or unsatisfiable (short: "UNSAT"). The "Modulo 
+  Theory" part means that the solver assumes a meaning for certain symbols in 
+  the formula. For instance the theory of integer arithmetic stipulates that the 
+  symbols `+`, `-, `*`, etc. have their regular everyday mathematical 
+  meaning.
+  When the formula is satisfiable, the SMT solver can also return a model for 
+  the formula. I.e. an assignment of the formula's variables that makes the 
+  formula evaluate to "true". For instance, on the formula "x > 5 /\ x = y * y", 
```

Comment:
uh, good point! Changing ..

Reply:
hm, after trying out:
 - in regular docs, the `{math}...` syntax works nicely -- using that in the future; the `$` syntax doesn't seem to work (not compiled as math..)
 - in the glossary, neither syntax seems to work -- I suspect because the glossary is a special environment; I didn't quickly find a workaround, so I'm leaving things here for now -- if/when we find a workaround, we should change it of course..